### DISCUSSION TITLE
Cannot connect Azure smtp to Supabase

### DISCUSSION THREAD

**Discussion by GaushJMeta** [supabase/supabase#20312] [Questions]
Hello All, I am not able to connect azure smtp to supabase , my port number is 587 and host is smtp.azurecomm.net and I have setup the username and password but I am getting this error '{
    "message": "Project <project ref> is not allowed to set up custom domain; please contact Supabase support."
}'. Any help would be appreciated.

**Comment by awalias** [comment]
I would try opening a ticket here and include the error message and project details: https://supabase.com/dashboard/support/new
